test_data/2015_portland_mayor_multi/2015_portland_mayor_multi_config.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
I removed this one. It's not a real thing (mayor is not multi-seat) and it doesn't test anything in particular.

Reply:
Oh that's true.  I added this because I wanted a short and simple multi seat test for debugging something, but yeah it doesn't need to be a test.